app/src/main/res/xml/preferences.xml

```diff
@@ -86,6 +86,16 @@
             android:title="@string/settings_disable_lockscreen_while_viewing_card"
             app:iconSpaceReserved="false"
             app:singleLineTitle="false" />
+
+        <SwitchPreferenceCompat
+            android:widgetLayout="@layout/preference_material_switch"
+            android:defaultValue="false"
+            android:key="@string/settings_key_use_volume_keys"
+            android:summary="@string/volume_keys_navigation_summary"
+            android:title="@string/volume_keys_navigation"
+            app:iconSpaceReserved="false"
+            app:singleLineTitle="false" />
```

Comment:
Just personal preference but I think I'd like this to be the first setting under "Cards" because it has the strongest effect and is most important to find for people with accessibility needs :)

Reply:
Updated that so "settings_key_use_volume_keys" is first on the list.